sds_data_manager/stacks/instrument_lambdas.py

```diff
@@ -102,20 +105,8 @@ def __init__(
         )
         rds_secret.grant_read(grantee=self.instrument_lambda)
 
-        # EventBridge Rule for this lambda
-        event_from_indexer_lambda = events.Rule(
-            self,
-            "EventFromIndexerLambda",
-            rule_name="event-from-indexer-lambda",
-            event_pattern=events.EventPattern(
-                source=["imap.lambda"],
-                detail_type=["Processed File"],
-                detail={
-                    "object": {"key": [{"exists": True}]},
-                },
-            ),
-        )
-
-        event_from_indexer_lambda.add_target(
-            targets.LambdaFunction(self.instrument_lambda)
-        )
+        # This sets up the lambda to be triggered by the SQS queue. Since this is a FIFO
+        # queue, each instrument will have messages processed in order. However,
+        # different instruments will be processed in parallel, with multiple instances
+        # of the batch_starter lambda.
```

Comment:
we can discuss about this outside of this PR. I feel like we can start with one event rule that hits SQS target and then add on as we need.